TextureExtraction tool/Program.cs

```diff
@@ -321,8 +320,15 @@ static void Main(string[] args)
                                         break;
                                     case "-tasks":
                                     case "-t":
-                                        i++;
-                                        options.Parallel.MaxDegreeOfParallelism = Int32.Parse(args[i]);
+                                        if (!int.TryParse(args[++i], out int parse))
+                                        {
+                                            Console.Error.WriteLine("Wrong syntax.");
```

Comment:
this is a good change, however we should point out that 'args[++i]' is not a valid value for -tasks.

Reply:
As in, for documentation? If so, I don't see the point, considering we didn't do so for 'args[i]' (unless I missed it somewhere)

Reply:
what i mean is if the user uses `-t -mip` because he sees that `-t` wants a value, we can communicate that with
$"Wrong syntax: \"{args[i-1]} {args[i]}\" Task needs a second Parameter".

Reply:
Ahhh, now I see what you're saying!